packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -171,13 +171,14 @@ const getRouterRenderer = ({
 	// so go ahead and assemble the full response body
 	const result = getHtml(
 		<Dom
-			basename={basename}
+			basename={appContext.basename}
```

Comment:
`basename` no longer a prop for `Dom` - still need to update implementation of `Dom` to remove `userAgent` prop as well - just read from `appContext`.